common/data_stores/data_store.py

```diff
@@ -65,3 +69,58 @@ def store_data(self, data, **kwargs):
             JSON: {status: "", matched_count: "", insert_id: ""}
         """
         return self._dsb.save_data(self._data_store_name, data, **kwargs)
+
+    def delete_data(self, data, **kwargs):
+        """Delete data from store.
+
+        Args:
+            data (json/bson): JSON / BSON string.
+
```

Comment:
write description for **kwargs

Reply:
Added statement for kwargs in Doc string.